src/com/lftechnology/java/training/alina/jdbc/controller/EmployeeController.java

```diff
@@ -0,0 +1,236 @@
+package com.lftechnology.java.training.alina.jdbc.controller;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.jdbc.constants.Constants;
+import com.lftechnology.java.training.alina.jdbc.dao.employee.impl.EmployeeDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.dao.user.impl.UserDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.dbutils.DbFacade;
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+import com.lftechnology.java.training.alina.jdbc.domain.EmployeeRole;
+import com.lftechnology.java.training.alina.jdbc.domain.User;
+import com.lftechnology.java.training.alina.jdbc.service.DateTimeService;
+import com.lftechnology.java.training.alina.jdbc.service.UserService;
+import com.lftechnology.java.training.alina.jdbc.service.UtilityService;
+
+/**
+ * Employee controller consists of functionalities regarding employee add, list, view ,delete
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class EmployeeController {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)